src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
Sure, no problem.  I'll update it now but am happy to wait on the PR to see if you figure out a better solution as well.